app/frontend/src/AppRoutes.tsx

```diff
@@ -162,28 +162,32 @@ export default function AppRoutes() {
       {
         // COMMUNITIES
       }
-      <AppRoute isPrivate path={communityRoute}>
-        <CommunityPage />
-      </AppRoute>
+      {process.env.REACT_APP_IS_COMMUNITIES_ENABLED && (
```

Comment:
So I/we can keep on pushing to `develop` without needing completely working code rather than needing a feature branch

Reply:
Do `=== "true"` or else this will be true if the env var is false (because it's a string)

Reply:
Hmm wouldn't it be easier to not set the environment variable at all rather than setting it to false explicitly (or more like `"false"` when parsed by JS) in that case? So we only set it if we want it to be truthy

Reply:
In theory, but it's easy to forget and have it set to false (ie. wasn't it set to false just before you changed it?)